Week5/Blockbuster/app/src/main/java/com/example/blockbuster/fragments/MovieFragment.kt

```diff
@@ -0,0 +1,45 @@
+package com.example.blockbuster.fragments
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import com.example.blockbuster.R
+import com.example.blockbuster.data.DataManager
+import com.example.blockbuster.data.Movie
+import kotlinx.android.synthetic.main.fragment_movie.*
+
+class MovieFragment : Fragment() {
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.fragment_movie, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        // Show details in the view
+        val model = activity?.let {  ViewModelProvider(it).get(DataManager::class.java) }
+        if (model != null) showMovieDetails(model.getMovieSelected())
+
+    }
+
+    private fun showMovieDetails(movie: Movie?){
+        /*
```

Comment:
I uncommented this code to see if I could get the images to load and I'm getting a security exception. I am unsure if Room can store images but that question may have been asked in the chat or you could ask it again, the Int for the drawable resources should've worked, and my security exception happens when I reload the app and may be an issue with using the URI. Sorry I don't have more information on this one but be sure to ask in the chat to get more suggestions from the larger community on the correct way to go about this.